File: BEE 1017.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        int hour = input.nextInt();
        int kmh = input.nextInt();
        double ans = (hour * kmh) / 12.0;
        System.out.printf("%.3f\n", ans);
    }
}

File: BEE 1017.cpp
#include<iostream>
#include<iomanip>
#define FIXED_FLOAT(x) std::fixed << std::setprecision(3) << (x)

using namespace std;

int main(){
    int hour, kmh;
    cin >> hour;
    cin >> kmh;
    double ans = (hour * kmh)/12.0;
    cout << FIXED_FLOAT(ans) << endl;
    return 0;
}